### PR TITLE
[EEN-14857] Files Sync/Re-Build

### PR DESCRIPTION
**Pull request by mrauer**
This takes care of the file(s) synchronization.

When using Skaffold, the files are sent to the api-docs pod, but in order to see the changes in your web browser, we also need to re-build the project.

Now, by default `run.sh` is started from the Dockerfile, and it does start the webserver. A new variable 
`EEN_ENVIRONMENT` has been introduced (default=prod). If that variable is set to `dev`, then any update in the `/source` directory will trigger a rebuild.

Can be tested via `tiamat project update apidocs`

Or with this sample skaffold.yaml:

```
apiVersion: skaffold/v1
build:
  artifacts:
  - context: .
    image: apidocs
    sync:
      infer:
      - '**/*'
deploy:
  helm:
    releases:
    - chartPath: ./chart/api-docs
      imageStrategy:
        helm: {}
      name: apidocs
      setValues:
        environment: dev
        imagePullPolicy: IfNotPresent
        ingress.enabled: "true"
        nodeSelector: ""
        replicaCount: 1
      values:
        image: apidocs
      valuesFiles:
      - ./chart/api-docs/values.yaml
kind: Config
```

### CODE REVIEW

**Comment by dailey-report** [comment]
:+1: 

suggest additional thumbs as I am not a Docker expert.

File: Dockerfile [hunk 1]
```diff
@@ -4,17 +4,21 @@ FROM ruby:2.6.5
...
+
 ADD default /etc/nginx/sites-available/default
 ADD nginx.conf /etc/nginx/nginx.conf
+
 # log files to standard out and error
 RUN ln -sf /dev/stdout /var/log/nginx/access.log && ln -sf /dev/stderr /var/log/nginx/error.log
-# start nginx
-CMD ["nginx", "-g", "daemon off;"]
+
+RUN chmod +x ./run.sh
```
Comment (georgekarrv) [inline Dockerfile]: it usually makes more sense to me to chmod the file and commit that into the repo

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Monitor file(s) update(s) in the background
+if [ "$EEN_ENVIRONMENT" != "prod" ]; then
```
Comment (EE-GSlomin) [inline run.sh]: Do we need to gate this watch script to ensure it's only running once? Possibly killing itself at the end of this script.

Comment (georgekarrv) [reply run.sh]: I don't think so, this is the entire life of the pod, if it's killed this pod dies. either way it is cleaned up

Comment (EE-GSlomin) [reply run.sh]: Definitely true for prod but not so much for development.

**Pull request merged**